tests/units/reporter/test__init__.py

```diff
@@ -84,65 +84,67 @@ def test__color_result(self, status: AntaTestStatus, expected_status: str) -> No
         assert report._color_result(status) == expected_status
 
     @pytest.mark.parametrize(
-        ("title", "number_of_tests", "expected_length"),
+        ("results_size"),
         [
-            pytest.param(None, 5, 5, id="all results"),
-            pytest.param(None, 0, 0, id="result for host1 when no host1 test"),
-            pytest.param(None, 5, 5, id="result for test VerifyTest3"),
-            pytest.param("Custom title", 5, 5, id="Change table title"),
```

Comment:
will probably need to add this back to be non breaking

Reply:
Implemented in test_titles below